### ISSUE TITLE
The select box falls behind the popup.

### ISSUE THREAD

**Issue by RustemHesenov** [octobercms/october#5593]
There is a small front-end error on the new plugin creation page. The select box falls behind the popup.

<img width="766" alt="Screen Shot 2021-05-10 at 11 19 05" src="https://user-images.githubusercontent.com/7926881/117620661-bf1a4700-b181-11eb-84e0-4ff11aa3249d.png">

**Comment by daftspunk** [comment]
Hi, thanks for the report. This bug was introduced in a recent build (v2.0.13) and will be fixed in the next one (v2.0.14).

As a workaround, you can roll back the system module to v2.0.12

```
composer require october/system 2.0.12
```

But don't forget to swap it back when v2.0.14 is released

```
composer require october/system "^2.0"
```

**Comment by daftspunk** [comment]
Thanks for the report. This is fixed in v2.0.14
